.circleci/config.yml

```diff
@@ -113,7 +113,7 @@ jobs:
       # Test
       - run:
           name: Run All Unit Tests
-          command: yarn nx run-many --target=test --all
+          command: yarn nx run-many --target=test --all --parallel=1
```

Comment:
Needed to add this to make make the circle ci happy.
Otherwise, the following errors were thrown randomly.

```
> nx run component:test 
Nx could not find process's output. Run the command without --parallel.

> nx run component-store:test 
Nx could not find process's output. Run the command without --parallel.
```

I was under the impression that this was handled by the migration by NX.
![image](https://user-images.githubusercontent.com/28659384/144245683-efa812ce-fa74-4f7c-bdf3-18fe5c17b75c.png)


@brandonroberts this could maybe be a bug in NX?

Reply:
Yea, if it's set in the `nx.json`, it shouldn't be necessary to manually set it everywhere for CI